src/main/java/org/prebid/server/proto/openrtb/ext/request/mgid/ExtImpMgid.java

```diff
@@ -0,0 +1,34 @@
+package org.prebid.server.proto.openrtb.ext.request.mgid;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+import java.math.BigDecimal;
+
+/**
+ * Defines the contract for bidRequest.imp[i].ext.mgid
+ */
+@AllArgsConstructor(
+        staticName = "of"
+)
+@Value
+public class ExtImpMgid {
+    @JsonProperty("accountId")
+    String accountId;
+
+    @JsonProperty("placementId")
+    String placementId;
+
+    @JsonProperty("cur")
```

Comment:
We don't need this annotation since Jackson json mapper uses `cur` field name by default.
Also, check all similar cases in this class (mapper uses snake-case strategy).